packages/simcore-sdk/src/simcore_sdk/node_ports_common/r_clone_utils.py

```diff
@@ -44,6 +44,7 @@ class _RCloneSyncTransferringMessage(_RCloneSyncMessageBase):
     _RCloneSyncTransferCompletedMessage,
     _RCloneSyncUpdatedMessage,
     _RCloneSyncTransferringMessage,
+    _RCloneSyncMessageBase,
```

Comment:
Q: By including the base class, does this mean that you are receiving "unclassified" messages?

Reply:
Correct, that's the case, there is a message which has exactly the same fields as the base class.